### ISSUE TITLE
Low Kitti RMSE results

### ISSUE THREAD

**Issue by kesaroid** [intel-isl/DPT#6]
Hi, I used your pre-trained model to obtain results for Kitti validation and the RMSE values are extremely high even after post-processing the depth results like you have done to visualize the images. 
All the depth pixels are way higher than expected, plus I noticed an inverse relationship where the closer objects have a higher value than the farther ones. 
I was hoping to get more insight from you on how to get the same RMSE values for Kitti as you have mentioned in your paper.

**Comment by ranftlr** [comment]
The released models are the general-purpose models that are affine invariant. They are arbitrarily scaled with respect to the true metric depth, so you can't directly evaluate RMSE on these models. They also predict inverse depth, not depth, which explains the inverse relationship that you observe.

The RMSE numbers in the the paper refer to models that have been finetuned on the respective datasets. I've just updated the code and uploaded the weights. You can download the weights for KITTI here: 

https://github.com/intel-isl/DPT/releases/download/1_0/dpt_hybrid_kitti-cb926ef4.pt

Then run with 
```
python run_monodepth.py -t dpt_hybrid_kitti --kitti_crop
```

**Comment by kesaroid** [comment]
Thank you. This was very helpful. I was able to reduce it to an RMSE of 2.80
Perhaps you might know how to reduce it further?
